src/ResourceManager/Sql/Commands.Sql/help/Stop-AzureRmSqlElasticPoolActivity.md

```diff
@@ -0,0 +1,171 @@
+﻿---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version:
+schema: 2.0.0
+---
+
+# Stop-AzureRmSqlElasticPoolActivity
+
+## SYNOPSIS
+Cancels the asynchronous updates operation on the elastic pool.
+
+## SYNTAX
+
+```
+Stop-AzureRmSqlElasticPoolActivity [-ServerName] <String> [-ElasticPoolName] <String> [-OperationId <Guid>]
+ [-ResourceGroupName] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Stop-AzureRmSqlElasticPoolActivity** cmdlet cancels the asynchronous updates operation on the elastic pool.
+
+## EXAMPLES
+
+### Example 1: Cancel the asynchronous updates operation on the elastic pool
+```
+PS C:\> Stop-AzureRmSqlElasticPoolActivity -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -ElasticPoolName "ElasticPool01" -OperationId af97005d-9243-4f8a-844e-402d1cc855f5
+```
+
+OperationId     : af97005d-9243-4f8a-844e-402d1cc855f5
+ServerName      : Server01
+DatabaseName    : ElasticPool01
+State           : CANCELLED
+Operation       : UpdateLogicalElasticPool
+ErrorCode       :
+ErrorMessage    :
+ErrorSeverity   :
+StartTime       : 10/15/2017 02:49:42 PM
+EndTime         : 10/15/2017 02:49:43 PM
+PercentComplete : 
+```
+
+This command cancels the asynchronous updates operation on the elastic pool.
```

Comment:
same comment

Reply:
@jaredmoo , comments addressed in the new commit.